src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java

```diff
@@ -82,6 +89,9 @@
     /** Period literal. */
     private static final String PERIOD = ".";
 
+    /** inherit doc literal. */
+    private static final String INHERIT_DOC = "{@inheritDoc}";
```

Comment:
Yeah i was working on that.
https://github.com/checkstyle/checkstyle/pull/3926
Above is pull request for that, all asked changes have been done.
But until it gets merged i had to handle that case that is why added it.